etc/config.xml

```diff
@@ -5,7 +5,8 @@
             <general>
                 <enable>0</enable>
                 <show_country_only>0</show_country_only>
+                <available_web_sites></available_web_sites>
```

Comment:
jetzt ist natürlich die Frage ob das ein "null" oder ein "" ist? Du willst ja eigentlich ein null?

Reply:
if that works as intended, it can be merged

Reply:
hmm schon ja, ich kann aber auch nicht einfach `null` da reinschreiben, weil das dann doch ein string `"null"` wäre?

Reply:
Man kann aber eh nicht nichts auswählen im Adminpanel also passt so? Wenn ich mit magerun den Pfad lösche, funktioniert es so, wie ich mir das vorgestellt habe.